dts/common/skeleton.dtsi

```diff
@@ -9,4 +9,6 @@
 	#size-cells = <1>;
 	chosen { };
 	aliases { };
+	leds { };
```

Comment:
Can you explain this further, why do we need empty nodes.  It's not typical to add these in skeleton.dtsi

Reply:
Was needed initially, but somehow it has became superfluous now. Removing the commit.